bplan/assets/html/app/shared/directives/nav.html

```diff
@@ -14,22 +14,22 @@
         </a>
     </li>
     <li class="textinput" data-ng-class="{open: searchResultCount > 1 && searchstring}">
-        <form class="navbar-form navbar-left" ng-submit="getDataForSearch()" navbar-input-group" role="search">
+        <form class="navbar-form navbar-left navbar-input-group" ng-submit="getDataForSearch()" role="search">
             <div class="form-group">
                 <div class="input-group">
                     <input type="text" class="form-control" data-ng-model="searchstring" data-ng-change="removeTagOnBackspace()" placeholder="Adresse oder Bebauungsplan-Nr.">
-                    <div
-                        class="input-group-addon search-button"
+                    <button
+                        class="search-button"
                         data-ng-click="getDataForSearch()"
                         data-ng-class="{middle: searchstring}">
                         <i class="fas fa-search" aria-label="Adresse oder Bebauungsplan-Nr. suchen"></i>
-                    </div>
-                    <div
+                    </button>
+                    <button
```

Comment:
this and also probably the button above need to be div for some unknown reason. If you use button the "No entries found" text won't disappear when clicking the button. No idea why

Reply:
another issue: the `No entries found` text will move the search bar up a little

Reply:
can't reproduce first issue but maybe that's dues to changes i've made, if it's still happening let me know which browser